index.html

```diff
@@ -217,29 +217,26 @@ <h3>Full Name</h3>
 							<div class="custom-card">
 								<div class="avatar-wrapper">
 									<img
-										src="https://media.istockphoto.com/vectors/thumbnail-image-vector-graphic-vector-id1147544807?k=6&m=1147544807&s=612x612&w=0&h=8CXEtGfDlt7oFx7UyEZClHojvDjZR91U-mAU8UlFF4Y="
-										alt=""
+										src="assets/img-alina.jpg"
+										alt="Alina Olteanu"
 										class="avatar"
 									/>
 								</div>
 								<div>
-									<h3>Full Name</h3>
+									<h3>Alina-Nicoleta Olteanu</h3>
 									<p>
-										Lorem ipsum dolor sit amet consectetur, adipisicing elit.
-										Molestiae omnis nobis quo dicta culpa eum exercitationem
-										provident ex iure pariatur? Tempora sunt accusamus tempore
-										modi. Quidem, corrupti perspiciatis aliquid dicta cum odio
-										sapiente tempora quas incidunt adipisci alias praesentium et
-										excepturi error maxime aspernatur sint qui doloremque
-										distinctio. Ex, unde.
+										I am an ambitious and curious individual with a strong passion for the IT field, 
+										which I developed during high school. 
+										I am always eager to learn and grow, whether through formal education or hands-on experience, and I am committed to staying updated on the latest trends in technology.
+										I enjoy working in teams, as I believe collaboration is key to achieving innovative solutions.
 									</p>
 								</div>
 								<div class="contact">
-									<a href="#"
+									<a href="www.linkedin.com/in/alina-olteanu-2392a1318"
 										><img src="assets/linkedin.svg" alt="linkedin"
```

Comment:
Aici ne trebuie si protocolul in href, adica trebuie sa avem: 
`<a href="https://www.linkedin.com/in/alina-olteanu-2392a1318/">`